### ISSUE TITLE
Unexpected token in pascalcase dependency

### ISSUE THREAD

**Issue by sQuarecoW** [victorgarciaesgi/vue-chart-3#103]
## Describe the bug
Vue crashes:

```js
Failed to compile.

./node_modules/vue-chart-3/node_modules/pascalcase/index.js 25:42
Module parse failed: Unexpected token (25:42)
You may need an appropriate loader to handle this file type, currently no loaders are configured to process this file. See https://webpack.js.org/concepts#loaders
| export const pascalcase = (value, options = {}) => {
|   const input = toString(value);
>   const regex = options.punctuationRegex ?? PUNCTUATION;
|   const output = input ? camelcase(regex ? input.replace(regex, ' ') : input, options) : '';
|   return output ? output[0].toLocaleUpperCase(options.locale) + output.slice(1) : '';
```

# Remember, `vue-chart-3` is only an *adapter* to use `chart.js` with Vue easily. So any `chart.js` related issue or question will be dismissed

## To Reproduce

- `npm i  vue-chart-3`
- use the example found here: https://vue-chart-3.netlify.app/guide/usage/dynamic-data.html (minus the shuffle)
- vue crashes

```sh
npm list @vue/runtime-core

`-- vue-chart-3@3.1.1
  +-- @vue/runtime-core@3.2.30
  `-- @vue/runtime-dom@3.2.30
    `-- @vue/runtime-core@3.2.30 deduped
```

## Version of `vue-chart-3`
v3.1.1

## Version of Vue

- [x] Vue 3
- [ ] Vue 2
- [ ] Nuxt 2
- [ ] Nuxt 3

**Comment by victorgarciaesgi** [comment]
Hey! Sorry it was a mistake to update it in the previous build, should be fixed on `3.1.2`!

**Comment by sQuarecoW** [comment]
Tnx for the quick reply and solution!
